dbms/src/Storages/FormatVersion.h

```diff
@@ -156,10 +176,14 @@ inline const StorageFormatVersion & toStorageFormat(UInt64 setting)
         return STORAGE_FORMAT_V4;
     case 5:
         return STORAGE_FORMAT_V5;
+    case 6:
+        return STORAGE_FORMAT_V6;
     case 100:
         return STORAGE_FORMAT_V100;
+    case 101:
+        return STORAGE_FORMAT_V101;
     default:
-        throw Exception("Illegal setting value: " + DB::toString(setting));
+        throw Exception("Illegal setting value: {}", setting);
```

Comment:
you're constructing an instance with the `code=settings`, which is not expected.

It should be something like:
```suggestion
        throw Exception(ErrorCodes::LogicalError, "Illegal setting value: {}", setting);
```